packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -3214,6 +3215,7 @@ export const IncomingCallNotification: (props: IncomingCallNotificationProps) =>
 // @beta
 export interface IncomingCallNotificationProps {
     alertText?: string;
+    allowVideo?: boolean;
```

Comment:
consider making this more explicit what is happening, I dont' fully get what "allow" means here?
Suggestion
```ts
acceptOptions: {
  showAcceptWithVideo: boolean
}
```